Cargo.toml

```diff
@@ -16,42 +16,43 @@ version = "0.83.2"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [package.metadata.binstall]
-pkg-url = "{ repo }/releases/download/{ version }/{ name }-{ version }-{ target }.{ archive-format }"
 pkg-fmt = "tgz"
+pkg-url = "{ repo }/releases/download/{ version }/{ name }-{ version }-{ target }.{ archive-format }"
 
 [package.metadata.binstall.overrides.x86_64-pc-windows-msvc]
 pkg-fmt = "zip"
 
 [workspace]
 members = [
 	"crates/nu-cli",
-	"crates/nu-engine",
-	"crates/nu-parser",
-	"crates/nu-system",
 	"crates/nu-cmd-base",
+	"crates/nu-cmd-dataframe",
 	"crates/nu-cmd-extra",
 	"crates/nu-cmd-lang",
-	"crates/nu-cmd-dataframe",
 	"crates/nu-command",
-	"crates/nu-protocol",
+	"crates/nu-engine",
+	"crates/nu-parser",
 	"crates/nu-plugin",
-	"crates/nu_plugin_inc",
-	"crates/nu_plugin_gstat",
-	"crates/nu_plugin_example",
-	"crates/nu_plugin_query",
-	"crates/nu_plugin_custom_values",
-	"crates/nu_plugin_formats",
+	"crates/nu-protocol",
 	"crates/nu-std",
+	"crates/nu-system",
 	"crates/nu-utils",
+	"crates/nu_plugin_custom_values",
+	"crates/nu_plugin_example",
+	"crates/nu_plugin_formats",
+	"crates/nu_plugin_gstat",
+	"crates/nu_plugin_inc",
+	"crates/nu_plugin_query",
 ]
 
 [dependencies]
+nu-ansi-term = "0.49.0"
```

Comment:
In general I am a fan of minimal diffs, but for once formatting I don't have too strong opinions.

Can we keep `nu-ansi-term`/`reedline in their own paragraph separated section (as it doesn't seem to re-sort them).

Github's merge diff algorithm sometimes claims conflicts when neighboring lines change (maybe only if there are addition/deletions in the section) but just to avoid conflicts before the release may be worth keeping in its own block.